app/js/components/encounters/encounterForm.jsx

```diff
@@ -119,7 +100,12 @@ const Encounter = (props) => {
             >
               {
                 locationArray.map(locations => (
-                  <option key={locations.uuid} value={locations.uuid}>{locations.display}</option>
+                  <option
+                    key={locations.uuid}
```

Comment:
Shouldn't this variable name be location?

Reply:
Corrected